adapters/Unity/UnityAdapter/GADMAdapterUnityRewardedAd.m

```diff
@@ -14,32 +14,34 @@
 
 #import "GADMAdapterUnityRewardedAd.h"
 #import "GADMAdapterUnityConstants.h"
-#import "GADMAdapterUnitySingleton.h"
+#import "GADUnityError.h"
 #import "GADMAdapterUnityUtils.h"
 
 @interface GADMAdapterUnityRewardedAd () <GADMAdapterUnityDataProvider, UnityAdsExtendedDelegate> {
-  // The completion handler to call when the ad loading succeeds or fails.
-  GADMediationRewardedLoadCompletionHandler _adLoadCompletionHandler;
-  // Ad configuration for the ad to be rendered.
-  GADMediationAdConfiguration *_adConfiguration;
-
-  // An ad event delegate to invoke when ad rendering events occur.
-  id<GADMediationRewardedAdEventDelegate> _adEventDelegate;
-
-  /// Game ID of Unity Ads network.
-  NSString *_gameID;
-
-  /// Placement ID of Unity Ads network.
-  NSString *_placementID;
-
-  /// YES if the adapter is loading.
-  BOOL _isLoading;
-
-  /// UUID for Unity instrument analysis
-  NSString *_uuid;
-
-  /// MetaData for storing Unity instrument analysis
-  UADSMetaData *_metaData;
+    int impressionOrdinal;
+    int missedImpressionOrdinal;
```

Comment:
we don't needs those any logic around.

Reply:
Sure. Should I remove the UADSMediationMetaData configuration too? Or implement it just without the above?

Reply:
let's remove UADSMediationMetaData for now. I'm checking if need it.